File: dotnet/cafdemalihapa/Directories/Hosting/Feature/Automations/AutomationsDirectory.cs
namespace cafdemalihapa.Directories.Hosting.Feature.Automations
{
    public class AutomationsDirectory()
    {
        public void Create()
        {
            var path = GetPath();
            Directory.CreateDirectory(path);
        }
        public string GetPath()
        {
            var directory = FeatureDirectory.GetPath();
            var automationsDirectory = Path.Combine(directory, "automations");
            return automationsDirectory;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Hosting/Feature/Automations/AutomationsDirectory.fs
namespace cafdemwimu.console.Directories.Hosting.Feature.Automations

open System.IO
open cafdemwimu.console.Directories.Hosting.Feature

type AutomationsDirectory() =
    member this.Create() =
        let path = this.GetPath()
        Directory.CreateDirectory(path) |> ignore

    member _.GetPath() =
        let directory = FeatureDirectory.GetPath()
        let automationsDirectory = Path.Combine(directory, "automations")
        automationsDirectory
